README.md

```diff
@@ -268,6 +268,35 @@ keep it.
 Use [proofit404/blacken](https://github.com/proofit404/blacken).
 
 
+### PyCharm
+
+(Adapted from [*Pylinting with PyCharm*](https://kirankoduru.github.io/python/pylint-with-pycharm.html))
+
+1. Install `black`.
```

Comment:
nit: In all of the docs when we refer to the project, we say `*Black*`, not <tt>\`black\`</tt>.